rai/core_test/ledger.cpp

```diff
@@ -1464,11 +1464,11 @@ TEST (ledger, inactive_supply)
 		ledger.process (transaction, send);
 	}
 	rai::transaction transaction (store.environment, nullptr, false);
-	ASSERT_EQ (10, ledger.supply (transaction));
-	ledger.inactive_supply = 60;
-	ASSERT_EQ (0, ledger.supply (transaction));
-	ledger.inactive_supply = 0;
-	ASSERT_EQ (50, ledger.supply (transaction));
+	ASSERT_EQ (10, ledger.supply.circulating_get (true));
+	ledger.supply.inactive_set (60);
+	ASSERT_EQ (0, ledger.supply.circulating_get (true));
+	ledger.supply.inactive_set (0);
+	ASSERT_EQ (50, ledger.supply.circulating_get (true));
 }
```

Comment:
Could you add a test here testing the cache? e.g.

```c++
ledger.inactive_supply = 60;
ASSERT_EQ (50, ledger.supply.circulating_get (false));
```

Reply:
To clarify, I think it should be an addition to this test.

Reply:
Good idea, added a separate test for the supply cache